packages/calypso-e2e/src/lib/flows/cancel-purchase-flow.ts

```diff
@@ -0,0 +1,26 @@
+import { Page } from 'playwright';
+
+type CancelReason = 'Another reason…';
+
+/**
+ *
+ */
+export async function cancelPurchaseFlow(
+	page: Page,
+	feedback: {
+		reason: CancelReason;
+		customReasonText: string;
+	}
+) {
+	await page
+		.getByRole( 'combobox', { name: 'WHY WOULD YOU LIKE TO CANCEL?' } )
+		.selectOption( feedback.reason );
+
+	await page
+		.getByRole( 'textbox', { name: 'CAN YOU PLEASE SPECIFY?' } )
+		.fill( feedback.customReasonText );
+
+	await page.getByRole( 'button', { name: 'Submit' } ).click();
+
+	await page.getByRole( 'button', { name: /Submit and (remove|cancel) plan/ } ).click();
```

Comment:
This is a new format of Flow type POM I'm trying out. This has some advantages:
- less boilerplate demanded by a class.
- ability to group multiple related flows under the same file.